mysql-test/suite/component_percona_telemetry/t/delete_obsolete_file.test

```diff
@@ -0,0 +1,40 @@
+# Test that obsolete files from another servers are removed,
+# but files with names not conforming to expected pattern are not.
+
+--source include/have_percona_telemetry.inc
+--source include/force_restart.inc
```

Comment:
Do we really need this as we are doing restart with custom telemetry parameters anyway?

Reply:
The idea here was driven by the fact that MTR framework doesn't restart mysqld between tests if not necessary. Here we change the default times for telemetry gathering, so restarting would reset these times to the defaults. But on the other hand having shorter times should not cause any problems.